stages/org.osbuild.coreos.platforms

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python3
+"""
+Process all necessary operations related to CoreOS
+platforms is crucial and specific to CoreOS. This step
+is essential for CoreOS exclusively.
+"""
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "platform": {
+      "description": "The artifactory platform name",
+      "type": "string"
+    },
+    "platforms_json": {
+        "description": "The path where platforms.json is stored",
+        "type": "string"
+    },
+    "bootpath": {
+      "type": "string",
+      "description": "The mounted location of the boot filesystem tree where the BLS entries and grubconfig  will be under ./loader/entries/*.conf",
+      "pattern": "^(mount|tree):\/\/\/",
+      "examples": ["tree:///boot", "mount:///", "mount:///boot"],
+      "default": "tree:///boot"
+    },
```

Comment:
I'm similarly wondering if we make this optional and just pick things up from what is defined by the user.

If no mounts exist then use the tree. If mounts exist then use the mountsroot.

In our current plans I know we want to just run this against mounts, but I could see possibly wanting to run this against a tree at some point - maybe.. I'll have to think about it some more.